pkgs/development/python-modules/pyworld/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib
+, buildPythonPackage, fetchPypi
+, numpy, cython
+}:
+
+buildPythonPackage rec {
+  pname = "pyworld";
+  version = "0.2.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b25bc84efb1bfe1fa0e10b19b391ef40b6e1a330a8e31d676befa58614880bb3";
+  };
+
+  nativeBuildInputs = [
+    cython
+    stdenv.cc
```

Comment:
This should't be needed, `stdenv.cc` should be available in each build environment (unless `stdenvNoCC` is used).